### ISSUE TITLE
Platform document edit from Platform Content Team

### ISSUE THREAD

**Issue by laucon** [department-of-veterans-affairs/va.gov-team#79501]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

## About your team

- **Team name**: `Platform Content Team`
- **Product name:** `[Modernized VA.gov Accessibility Strategy document](https://vfs.atlassian.net/wiki/spaces/GS/pages/2374664350/Modernized+VA.gov+Accessibility+Strategy)`
- **Product manager:**  `Joe Knipes`
- **Slack channel:** `[#platform-content](https://join.slack.com/share/enQtNjkwMDUyMTAxMDMyMC0wNWQ0Mzk2M2UwMDdjOGM0MmM4NTBmYjNjZWQ4ZDg0YmFjM2EzNTkxYmZjOGVmZDdhZGQxZTAxZjBhYTZlY2U0)`
- **Dedicated content writer on your team (if you have one):** `Lauren Conrad`

 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

Right now, the Platform Content team is looking at Platform documents that link to google docs, google sheets, google slides, and google drive folders and editing them so that those links are no longer in those documents. [Modernized VA.gov Accessibility Strategy](https://vfs.atlassian.net/wiki/spaces/GS/pages/2374664350/Modernized+VA.gov+Accessibility+Strategy) has 9 such links. I highlighted all 9 links in [Modernized VA.gov Accessibility Strategy](https://vfs.atlassian.net/wiki/spaces/GS/pages/2374664350/Modernized+VA.gov+Accessibility+Strategy). We need to go one by one and get rid of the links. In some cases, that might mean describing a google sheet that you guys use and saying "message so-and-so team get access to this sheet." Or maybe it might mean transforming a google doc into either a platform doc or a github doc and linking to that.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?

We're flexible, but we would love to know what sprint could be a deadline for this project. Depending on how much is on your plate already, maybe we could make a deadline for this as the end of this sprint, being April 10. We could also have the deadline be the end of the following sprint, which would be April 24. So we're flexible, but we'd love to have a clear timeline in mind. 

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

## Supporting artifacts
Here is the confluence version of the document: https://vfs.atlassian.net/wiki/spaces/GS/pages/2374664350/Modernized+VA.gov+Accessibility+Strategy

Here is the Platform website version of the document: https://depo-platform-documentation.scrollhelp.site/getting-started/modernized-va-gov-accessibility-strategy

**Comment by coforma-jamie** [comment]
After reviewing the document, we determined that the Google Docs/Sheets in question are owned by Governance and are in constant use. We advised Lauren to speak with the Governance team about this, instead of CAIA. ([Slack thread](https://dsva.slack.com/archives/CL181NRJQ/p1712062799765229))
